client/scss/layouts/_login.scss

```diff
@@ -1,18 +1,28 @@
 // overrides default nice padding defined in variables.scss
 $desktop-nice-padding: 100px;
 
-.login,
-.login .content-wrapper {
-  background-color: $color-grey-1;
-  color: $color-grey-3;
-}
-
 .login {
-  margin-inline-start: 0;
-  height: 100%;
+  background: theme('colors.primary.DEFAULT');
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0px 390px 170px;
+  gap: 10px;
+  position: relative;
 
   .content-wrapper {
-    border: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 30px;
+    isolation: isolate;
+    background: none;
+    border: none;
+    flex: none;
+    order: 6;
```

Comment:
It's not good practice to use order to re-order content if it can be avoided. It creates a problem for accessibility where the focus order does not match the visual order. There are exceptions to this guide but it's best to get the DOM in the order we want first.